subsys/bluetooth/shell/gatt.c

```diff
@@ -946,43 +949,73 @@ static int cmd_unregister_test_svc(const struct shell *sh,
 	return 0;
 }
 
-static void notify_cb(struct bt_conn *conn, void *user_data)
+static uint8_t found_attr(const struct bt_gatt_attr *attr, uint16_t handle,
+			  void *user_data)
 {
-	const struct shell *sh = user_data;
+	const struct bt_gatt_attr **found = user_data;
 
-	shell_print(sh, "Nofication sent to conn %p", conn);
+	*found = attr;
+
+	return BT_GATT_ITER_STOP;
+}
+
+static const struct bt_gatt_attr *find_attr(uint16_t handle)
+{
+	const struct bt_gatt_attr *attr = NULL;
+
+	bt_gatt_foreach_attr(handle, handle, found_attr, &attr);
+
+	return attr;
```

Comment:
Should we consider adding a `bt_gatt_find_by_handle` function?

Reply:
Yes, I think so. It's already implemented as the internal function `find_attr` in `gatt.c`.